src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/tests/test_sas_token_auth.py

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+#pylint: disable=line-too-long
+
+import unittest
+from azure.cli.command_modules.iot.sas_token_auth import SasTokenAuthentication
+
+
+class TestSasTokenAuth(unittest.TestCase):
+    def test_generate_sas_token(self):
+        # Prepare parameters
+        uri = 'iot-hub-for-test.azure-devices.net/devices/iot-device-for-test'
+        policy_name = 'iothubowner'
+        access_key = '+XLy+MVZ+aTeOnVzN2kLeB16O+kSxmz6g3rS6fAf6rw='
```

Comment:
>+XLy+MVZ+aTeOnVzN2kLeB16O+kSxmz6g3rS6fAf6rw= [](start = 22, length = 44)

is this a security issue for someone else reading the repo?  Could they access your resources with this key?